stories/core/DropdownMenu.stories.tsx

```diff
@@ -264,3 +265,36 @@ Submenu.parameters = {
     },
   } as CreeveyStoryParams,
 };
+
+export const WithSeparator: Story<DropdownMenuProps> = (args) => {
+  const { menuItems, ...rest } = args;
+  const onClick = (index: number, close: () => void) => () => {
+    action(`Item #${index} clicked!`)();
+    close();
+  };
+  const dropdownMenuItems = (close: () => void) => [
+    <MenuItem key={1} onClick={onClick(1, close)}>
+      Item #1
+    </MenuItem>,
+    <MenuItem key={2} onClick={onClick(2, close)}>
+      Item #2
+    </MenuItem>,
+    <MenuDivider key={3} />,
+    <MenuItem key={4} onClick={onClick(3, close)} disabled>
+      Item #3
+    </MenuItem>,
+    <MenuItem key={5} onClick={onClick(4, close)}>
+      Item #4
+    </MenuItem>,
+  ];
+  return (
+    // Body height is the same as Select component height therefore clicking outside would not close dropdown.
+    <div style={{ minHeight: 200 }}>
```

Comment:
Can you move this div into a decorator? It's adding noise to the story

Reply:
I have tried that previously and I can't because it breaks `Docs` tab:
![image](https://user-images.githubusercontent.com/36186912/134629523-f2a80839-0331-46be-b4e2-bd9263d004f2.png)

Reply:
Try this?
https://github.com/storybookjs/storybook/issues/12596#issuecomment-723440097

Reply:
Changed.